src/frontend/components/DashboardLive/index.spec.tsx

```diff
@@ -477,6 +478,37 @@ describe('components/DashboardLive', () => {
     );
   });
 
+  it('check display title and starting date on the title bar ', async () => {
+    render(
+      wrapInVideo(
+        wrapInIntlProvider(
```

Comment:
If you are using the `render` from our utils, it manages to wrap the component you want to render in the intl provider. I think you don't need it.

Reply:
Ah oui en effet.